.murdock

```diff
@@ -73,6 +73,7 @@ z1
 : ${TEST_KCONFIG_ENFORCE_APP_GROUPS:="
 tests/cb_mux*
 tests/congure_*
+tests/disp_dev
```

Comment:
I think this is not needed, just by having an `app.config.test` it will get tested.

Reply:
Removed (and also removed `tests/touch_dev` from the list.